.github/workflows/publish.yml

```diff
@@ -59,7 +59,6 @@ jobs:
       # Before publishing, we want to test the extension against whatever
       # version of the scanner is currently live.
       use-scanner-tarballs: false
-    secrets: inherit
```

Comment:
I think this might actually be necessary, so I recommend leaving it in.

Reply:
As discussed offline, this was something I added to fetch the internal repo. We can remove this.